src/geo/ui/fullscreen.js

```diff
@@ -84,57 +68,49 @@ cdb.ui.common.FullScreen = cdb.core.View.extend({
       }
 
       if (mapView) {
-
         if (this.model.get("allowWheelOnFullscreen")) {
           mapView.options.map.set("scrollwheel", true);
         }
-
       }
-
     } else {
-
       cancelFullScreen.call(doc);
-
     }
   },
 
   render: function() {
-    if (this._canFullScreenBeEnabled()) {
-      var $el = this.$el;
-      var options = _.extend(this.options);
-      $el.html(this.options.template(options));
-    } else {
-      cdb.log.info('FullScreen is deprecated on insecure origins. See https://goo.gl/rStTGz for more details.');
+    var $el = this.$el;
```

Comment:
This variable is only being used once. We could use this.$el directly in line #88.